itests/fevm_test.go

```diff
@@ -58,7 +58,7 @@ func buildInputFromuint64(number uint64) []byte {
 // recursive delegate calls that fail due to gas limits are currently getting to 229 iterations
 // before running out of gas
 func recursiveDelegatecallFail(ctx context.Context, t *testing.T, client *kit.TestFullNode, filename string, count uint64) {
-	expectedIterationsBeforeFailing := int(229)
+	expectedIterationsBeforeFailing := int(220)
```

Comment:
Mild worry that this test as written is highly sensitive to gas changes. They are in flux these days as we close down the FEVM release (and will stabilise from now on), however, we will need to keep adjusting this test with further gas changes down the line.

Reply:
> Mild worry that this test as written is highly sensitive to gas changes

It's annoying, but I'd like to keep it as it tells us when we're changing things.

> In the future, we should be OK with ratching up this number, but we absolutely cannot decrease it as it may break existing contracts.

This is the same as any other gas related thing. If you're contract makes recursive calls >200 deep, don't.

In general, changing gas values can break any contract.